### ISSUE TITLE
Fastboot deprecation

### ISSUE THREAD

**Issue by oskarrough** [ronco/ember-cli-head#29]
Hi, seeing this with ember-cli-fastboot 1.0.0 rc.2 and ember-cli-head 0.2.2.

```bash
DEPRECATION: Having fastboot specific code in app directory of 
/Users/xxx/Sites/xxx/node_modules/ember-cli-head is deprecated. Please move it to fastboot/app directory.
```

I think it is this https://gist.github.com/kratiahuja/d22de0fb1660cf0ef58f07a6bcbf1a1c#addon-containing-fastboot-based-initializers-as-appinstance-initializersfastbootjs

Also see https://github.com/ember-fastboot/ember-cli-fastboot/issues/387#issuecomment-304445510

**Comment by kratiahuja** [comment]
This is already fixed with : https://github.com/ronco/ember-cli-head/pull/21 Can you provide a reproduction?

**Comment by kratiahuja** [comment]
Deprecation path seems wrong too: /Users/xxx/Sites/xxx/node_modules/ember-cli-head

**Comment by oskarrough** [comment]
The `xxx` part is my username. Or is the path otherwise wrong?

Here's a reproduction: https://github.com/oskarrough/fastboot-head-test. Thank you!

**Comment by kratiahuja** [comment]
The path is wrong.

**Comment by kratiahuja** [comment]
That is just a warning since we are doing the auto migration for you. Once addons and fastboot drop support for pre fastboot 1.0, they can address this deprecation. 

Your app would still work.

**Comment by oskarrough** [comment]
It does work :) I just wanted to help fix the deprecation.

**Comment by cibernox** [comment]
@oskarrough I believe I closed this in #32

**Comment by oskarrough** [comment]
@cibernox confirmed. Thanks!
